Data/Sys/GameSettings/GP7E01.ini

```diff
@@ -116,38 +655,351 @@ $Board - Neon Heights: Balance Changes
 044e2564 3b200014
 044e269c 3b000014
 e2000001 80008000
-*This cheat makes the chests now cost 20 coins, and Bob-omb/nothing chests now contain 20 coins.
+202f2f3c 0000007d
+28527bfa 00000256
+04527c78 4039999a
+e2000002 80008000
+*All chests cost 20 coins, and the non-star chests both contain 20 coins. The shooting board event now awards 10 coins, but only if the player hits all the targets. It is now possible for 1 player to reach the star in the rocket happening event.
 
-$Board - Neon Heights: Bowser Path for Free
-204FDEF4 2C03000A
-044FDEF4 2C030000
-044FDF5C 38A00000
-E2000001 80008000
+$Board - Neon Heights: Bowser Path for Free [gamemasterplc]
+204fdef4 2c03000a
+044fdef4 2c030000
+044fdf5c 38a00000
+e2000001 80008000
 *This cheat makes that path Koopa Kid is blocking not cost any coins to traverse.
 
-$Board - Pagoda Peak: Bottle Rocket Launch Always Succeeds
-204DF8A0 2803004B
-044DF8A4 60000000
-E2000001 80008000
+$Board - Pagoda Peak: Bottle Rocket Launch Always Succeeds [gamemasterplc]
+204df8a0 2803004b
+044df8a4 60000000
+e2000001 80008000
 *This cheat makes the Pagoda Peak bottle rocket always go towards the top.
 
-$Board - Pyramid Park: Free Chomp Rides
-204E10B4 7C03D000
-044E10B0 386003E7
-044E1114 386003E7
-044E14EC 386003E7
-044E17A4 38A00000
-E2000001 80008000
+$Board - Pyramid Park: Chomps Sell Stars, Players Start with 0 Stars
+204ddf34  4180ffb8
```

Comment:
There are two spaces between each column in this code. Is this intended?

Reply:
nope